### ISSUE TITLE
Wrong error message and auto-fix message for unfinished enum declaration

### ISSUE THREAD

**Issue by mellinoe** [OmniSharp/omnisharp-vscode#284]
Consider this file, which doesn't compile but is only missing a couple of braces for the enum declaration:

``` CSharp
namespace Example
{
    public interface X
    {
        object GetY(Z z);
    }

    public enum Z
}
```

(renamed stuff in the above to be simpler).

I get some weird messages from this:
![image](https://cloud.githubusercontent.com/assets/8918977/15079900/15f5e3c8-1370-11e6-9e58-19e13c8254c5.png)

`{ expected [netstandard1.5]`
Doesn't seem to make sense.

Also, the "auto-fix suggestion" is also wrong:
![image](https://cloud.githubusercontent.com/assets/8918977/15079923/3f719dfa-1370-11e6-92c3-dcd393d44045.png)

`Delegate invocation can be simplified.`

**Comment by mellinoe** [comment]
It seems like the `Delegate invocation can be simplified` message pops up in other random places too. For example, if I have an unused using statement, the first auto-correct suggestion is `Delegate invocation can be simplified`, and the second is `Remove Unused Usings`.

**Comment by DustinCampbell** [comment]
This should be fixed in the latest release (1.5.2)
